tests/zero_code_change/pt_utils.py

```diff
@@ -7,22 +7,34 @@
 import torch.nn.functional as F
 import torchvision
 import torchvision.transforms as transforms
+from packaging import version
 
 
 def get_dataloaders() -> Tuple[torch.utils.data.DataLoader, torch.utils.data.DataLoader]:
     transform = transforms.Compose(
         [transforms.ToTensor(), transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))]
     )
 
+    # Temporary Change to allow the test to run with pytorch 1.7 RC3
+    # Smdebug breaks when num_workers>0 for Pytorch 1.7.0
+    if version.parse(torch.__version__) >= version.parse("1.7.0"):
```

Comment:
nit: Shouldn't this check the version of torchvision instead of torch since you've already identified that the issue is caused by a dataloader coming from torchvision?

Reply:
The dataloader is defined in Torch `torch.utils.data.DataLoader`.